reflectutil/setter.go

```diff
@@ -155,6 +162,8 @@ func valueFor(typ reflect.Type, tag reflect.StructTag, value interface{}) reflec
 		valueStr = x.Sel.Name
 	case *protop.Literal:
 		valueStr = x.SourceRepresentation()
+	case uint8:
+		return reflect.ValueOf(x)
```

Comment:
this case stands out a bit from the rest. add a comment?

Reply:
yeah, that doesn't look right, will trying to make it more obvious.

Reply:
I removed this special case and return the whole slice of bytes instead.